### ISSUE TITLE
Añadir un `CONTRIBUTING.md` para los guidelines de las contribuciones y para definir el estilo

### ISSUE THREAD

**Issue by AleixMT** [AleixMT/Problemas-Computadores#33]
El `CONTRIBUTING.md` es un archivo "estándar" de los repos github donde se explica como se debe proceder para colaborar con un determinado repositorio. Ahí se explican las reglas para hacerlo, si se usa alguna guía de estilo, que tests o convencionies debe cumplir el código para poderse añadir... 

También serviria para quitar alguna explicación innecesaria en el `README.md` principal, ya que hay mucho texto y preferiria que fuera especificamente para alumnos, mientras que el `CONTRIBUTING.md` es para colaboradores.

**Comment by AreyFerreroRamos** [comment]
Me parece muy buena idea separar la información exclusiva para profesores de la información para los alumnos. Entiendo que sólo habrá un `CONTRIBUTING.md` para todo el repo, no? Adelante con ello.

**Comment by AleixMT** [comment]
Sí, solo habría uno en la raíz del repo.

Me pondré a ello en cuanto pueda 💪

**Comment by AreyFerreroRamos** [comment]
Perfecto. Cerramos el issue entonces?

**Comment by AreyFerreroRamos** [comment]
No cerramos el issue porque no está completada la funcionalidad. Me queda claro.

**Comment by AleixMT** [comment]
Sí, en cuanto tengamos la PR fusionada entonces cerramos el issue.
